### DISCUSSION TITLE
Is it possible to specify a path for the application.yml?

### DISCUSSION THREAD

**Discussion by driwand** [lavalink-devs/Lavalink#626] [Q&A] [answered]
Hello, I am wondering if its possible to something as the following:

> java -jar Lavalink.tar /path/to/application.yml

**Comment by freyacodes** [comment, marked as answer]
Yep 

```
java -jar Lavalink.jar --spring.config.location=/path/to/application.yml
```

The config system is very flexible. Full docs: https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#features.external-config.files
